pandas/io/tests/test_pytables.py

```diff
@@ -3806,6 +3806,17 @@ def f():
             result = store.select_column('df3', 'string', start=-2, stop=2)
             tm.assert_almost_equal(result.values, df3['string'].values[-2:2])
 
+            # GH 10392 - make sure column name is preserved
+            df4 = DataFrame({'A': np.random.randn(10), 'B': 'foo'})
+            store.append('df4', df4, data_columns=True)
+            expected = df4['B']
+            try:
+                result = store.select_column('df4', 'B')
+            except KeyError as err:
```

Comment:
Ok. I just wanted to change the error type so that it would be clear it is one of the things being tested, not an error in the testing code itself.